### PR TITLE
Add _allBillingReceiptsSum

### PR DESCRIPTION
**Pull request by toplenboren**
We need this api endpoint for Bank Analytics REST API

### CODE REVIEW

File: apps/condo/domains/billing/schema/SumBillingReceiptsService.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+                // If you try to add more periods to the where clause, like: where: (period=a1 period=a2) you will get an error
+                // If you try to be hacky and do this: where: (period=a1 period_gt=a1) you are going to have intersection of these clauses which means you will get only period=a1 ones!
+                if (!where.period || typeof where.period !== 'string') {
+                    throw new GQLError(ERRORS.PERIOD_BADLY_SPECIFIED)
+                }
+
+                const billingReceiptsLoader = new GqlWithKnexLoadList({
+                    listKey: 'BillingReceipt',
+                    fields: 'id toPay',
+                    where: where,
```
Comment (VKislov) [inline apps/condo/domains/billing/schema/SumBillingReceiptsService.js:55]:
```suggestion
                    where,
```

File: apps/condo/domains/billing/schema/SumBillingReceiptsService.test.js [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+ * Generated by `createservice billing.SumBillingReceiptsService`
+ */
+
+const { makeLoggedInAdminClient, makeClient, expectToThrowAuthenticationErrorToResult, expectToThrowAccessDeniedErrorToResult, expectToThrowGQLError } = require('@open-condo/keystone/test.utils')
+
+const { sumBillingReceiptsByTestClient } = require('@condo/domains/billing/utils/testSchema')
+
+const { ERRORS } = require('./SumBillingReceiptsService')
+
+const { makePayer } = require('../../acquiring/utils/testSchema')
```
Comment (VKislov) [inline apps/condo/domains/billing/schema/SumBillingReceiptsService.test.js:11]:
```suggestion
const { makePayer } = require('@condo/domains/acquiring/utils/testSchema')
```

File: apps/condo/domains/billing/schema/SumBillingReceiptsService.test.js [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+const { makeClientWithSupportUser, makeClientWithNewRegisteredAndLoggedInUser } = require('../../user/utils/testSchema')
```
Comment (VKislov) [inline apps/condo/domains/billing/schema/SumBillingReceiptsService.test.js:12]:
```suggestion
const { makeClientWithSupportUser, makeClientWithNewRegisteredAndLoggedInUser } = require('@condo/domains/user/utils/testSchema')
```

File: apps/condo/domains/billing/schema/SumBillingReceiptsService.test.js [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+const { createTestBillingReceipt } = require('../utils/testSchema')
```
Comment (VKislov) [inline apps/condo/domains/billing/schema/SumBillingReceiptsService.test.js:13]:
```suggestion
const { createTestBillingReceipt } = require('@condo/domains/billing/utils/testSchema')
```

File: apps/condo/domains/billing/schema/SumBillingReceiptsService.test.js [hunk 5]
```diff
@@ -0,0 +1,123 @@
...
+                    message: ERRORS.PERIOD_BADLY_SPECIFIED.message,
+                },
+                'result'
+            )
+        })
+    })
+    describe('access checks', () => {
+        test('admin can sum payments', async () => {
+            const period = payers.singleReceipt.billingReceipts[0].period
+            const where = { context: { organization: { id: payers.singleReceipt.organization.id } }, period: period }
```
Comment (VKislov) [inline apps/condo/domains/billing/schema/SumBillingReceiptsService.test.js:94]:
```suggestion
            const where = { context: { organization: { id: payers.singleReceipt.organization.id } }, period }
```

File: apps/condo/domains/billing/utils/testSchema/index.js [hunk 6]
```diff
@@ -59,6 +60,8 @@ const { FLAT_UNIT_TYPE } = require('@condo/domains/property/constants/common')
 const { execGqlWithoutAccess } = require('@open-condo/codegen/generate.server.utils')
 const { buildFakeAddressAndMeta } = require('@condo/domains/property/utils/testSchema/factories')
 const { CONTEXT_FINISHED_STATUS } = require('@condo/domains/acquiring/constants/context')
+const {SUM_PAYMENTS_QUERY} = require("../../../acquiring/gql");
+const Big = require("big.js");
```
Comment (VKislov) [inline apps/condo/domains/billing/utils/testSchema/index.js:64]:
```suggestion
const Big = require("big.js")
```

**Review by VKislov — APPROVED** [review]

File: apps/condo/domains/billing/schema/SumBillingReceiptsService.js (hunk 1 above)
Comment (toplenboren) [reply apps/condo/domains/billing/schema/SumBillingReceiptsService.js:55]: Done!

File: apps/condo/domains/billing/schema/SumBillingReceiptsService.test.js (hunk 3 above)
Comment (toplenboren) [reply apps/condo/domains/billing/schema/SumBillingReceiptsService.test.js:12]: Done!

**Review by dkoviazin — APPROVED** [review]

**Pull request merged**